packages/lib/sync/services/index.ts

```diff
@@ -0,0 +1,7 @@
+import { ISyncServices } from "../ISyncService";
+import CloseComService from "./CloseComService";
+import SendgridService from "./SendgridService";
+
+const services: ISyncServices[] = [CloseComService, SendgridService];
```

Comment:
New sync services will need to extend `SyncServiceCore`, implement `ISyncService` and be added in this list.